### PR TITLE
document missing test coverage

### PR DESCRIPTION
**Pull request by grosser**
... prevents any new coverage gaps from being added

@mcrossen

### CODE REVIEW

File: test/helper.rb [hunk 1]
```diff
@@ -1,3 +1,8 @@
 require "bundler/setup"
-require "minitest/autorun"
 
+require "single_cov"
+SingleCov.setup :minitest
+
+require "maxitest/autorun"
+
+$LOAD_PATH << "lib"
```
Comment (grosser) [inline test/helper.rb:8]: makes tests run with just `ruby test/test_foo.rb`

File: test/helper.rb [hunk 2]
```diff
@@ -1,3 +1,8 @@
 require "bundler/setup"
-require "minitest/autorun"
 
+require "single_cov"
+SingleCov.setup :minitest
+
+require "maxitest/autorun"
```
Comment (grosser) [inline test/helper.rb:6]: fixes some common minitest annoyances and adds red/green output

File: test/test_deepsort.rb [hunk 3]
```diff
@@ -1,6 +1,8 @@
 require_relative "helper"
 require "deepsort"
 
+SingleCov.covered! file: "lib/deepsort.rb", uncovered:12
```
Comment (grosser) [inline test/test_deepsort.rb:4]: lots of uncovered code :(

**Comment by grosser** [comment]
travis still does not update ... https://travis-ci.org/mcrossen/deepsort
did you try enable/disable in travis UI ? https://travis-ci.org/account/repositories

**Comment by grosser** [comment]
@mcrossen 🏓

**Comment by mcrossen** [comment]
what do you mean by "travis still does not update"? It seems to be building the PRs. Do you mean that it won't push a new version of the gem? in that case I have to trigger it manually by tagging a commit. I'll do that after merging this.

**Comment by grosser** [comment]
travis status does not show on githubs PR screen or on the commit

**Comment by grosser** [comment]
Screen Shot 2019-05-07 at 6 07 04 PM

**Pull request merged**